.github/workflows/e2e.install.yaml

```diff
@@ -0,0 +1,52 @@
+name: E2E - Chart Install
+
+on:
+  pull_request:
+    paths:
+      - "**"
+      - "!**.md"
+  workflow_dispatch:
+
+jobs:
+  Install:
+    name: Chart Install
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Build Docker images for Controller Manager
+        uses: docker/build-push-action@v2.4.0
+        with:
+          file: config/dockerfiles/controller-manager/Dockerfile
+          tags: kubespheredev/devops-controller:e2e
+          push: false
+
+      - name: Build Docker images for APIServer
+        uses: docker/build-push-action@v2.4.0
+        with:
+          file: config/dockerfiles/apiserver/Dockerfile
+          tags: kubespheredev/devops-apiserver:e2e
+          push: false
+
+      - name: Build Docker images for Tools
+        uses: docker/build-push-action@v2.4.0
+        with:
+          file: config/dockerfiles/tools/Dockerfile
+          tags: kubespheredev/devops-tools:e2e
+          push: false
+
+      - name: Run E2E Test
+        uses: johnniang/skywalking-infra-e2e@main
```

Comment:
Why would I use repo `johnniang/skywalking-infra-e2e@main`? Because I have enhanced some features into the repo.

Before E2E testing is ready,  I prefer to use this `action uses`.